sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -68,7 +80,17 @@ public static Sampler alwaysOn() {
    * @since 0.1.0
    */
   public static Sampler alwaysOff() {
-    return ALWAYS_OFF;
+    return AlwaysOffSampler.INSTANCE;
+  }
+
+  /**
+   * Returns a {@link Sampler} that always makes a "no" decision on {@link Span} sampling.
```

Comment:
this javadoc needs to be corrected.

Reply:
Obsolete, decided to split into just the cleanup for the moment.